packages/components/src/components/PipelineRuns/PipelineRuns.js

```diff
@@ -35,6 +35,8 @@ import './PipelineRuns.scss';
 
 const PipelineRuns = ({
   cancelPipelineRun,
+  createPipelineRunUrl = urls.pipelineRuns.byName,
```

Comment:
let's do this using defaultProps instead, at the bottom of the file before the export:

```js
PipelineRun.defaultProps = {
  // here
};
```

this and propTypes will help big time when it comes to documenting the components for broader use

Reply:
Want me to add proptypes ?

Reply:
no just defaultProps will be fine for now. We need to discuss how we're going to document the component library, including patterns for propTypes and other tools that will help make it as painless as possible.

Reply:
discussed offline, issue with defaultProps so we'll go with this for now and revisit later as part of doc effort for the component library